docs/user_guide/3_getting_started.md

```diff
@@ -27,34 +40,50 @@ We have created a one-liner that is able to install Docker, download the image a
   ```
   In the above example, ‘enp0s25’ is the interface ID that is needed.
 
+* **Get AWS login credentials**
```

Comment:
You need to rewrite this section. It should be more related to ROS sending to Shadow rather than AWS. People should now care about it. It might be Google Cloud in 1 month.